src/riak_kv_http_cache.erl

```diff
@@ -10,15 +10,19 @@
 	 terminate/2,
 	 code_change/3]).
 
--define(SERVER, ?MODULE).
+-record(st,
+    {
+        ts :: undefined|erlang:timestamp(),
+        stats = []
+    }
+).
 
--record(st, {ts, stats = []}).
 
 start_link() ->
     gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
 
-get_stats() ->
-    gen_server:call(?MODULE, get_stats).
+get_stats(Timeout) ->
```

Comment:
Would be nice with either a type or a comment stating in which unit the Timeout is given.
`-spec get_stats(Milliseconds :: non_neg_integer()) -> .....` or similar

Reply:
spec added